File: 3931-check-adjacent-digit-differences/3931-check-adjacent-digit-differences.dart
class Solution {
  bool isAdjacentDiffAtMostTwo(String s) {
    for (int i = 1; i < s.length; i++) {
      if ((s.codeUnitAt(i) - s.codeUnitAt(i - 1)).abs() > 2) {
        return false;
      }
    }
    return true;
  }
}

File: 3931-check-adjacent-digit-differences/3931-check-adjacent-digit-differences.py
class Solution(object):
    def isAdjacentDiffAtMostTwo(self, s):
        """
        :type s: str
        :rtype: bool
        """
        for i in range(1, len(s)):
            if abs(ord(s[i]) - ord(s[i - 1])) > 2:
                return False
        return True